src/unicorn/mod.rs

```diff
@@ -20,10 +21,12 @@ pub mod dimacs_parser;
 pub mod emulate_loader;
 pub mod memory;
 pub mod optimize;
+pub mod quarc;
 pub mod qubot;
 pub mod sat_solver;
 pub mod smt_solver;
 pub mod unroller;
+pub mod z3solver_impl;
```

Comment:
Likewise for #[cfg(feature = "z3")] here.

Reply:
it works I've changed it :)